### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/pkgx#764] [Announcements]
## What's Changed
* disable PS1 mangling by @mxcl in https://github.com/pkgxdev/pkgx/pull/756
* mkdir before you install by @mxcl in https://github.com/pkgxdev/pkgx/pull/758

**Full Changelog**: https://github.com/pkgxdev/pkgx/compare/v1.0.0-alpha.4...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/pkgxdev/pkgx/releases/tag/v1.0.0'>1.0.0</a>.</em>

**Comment by magnusviri** [comment]
Congrats!
